src/app/home/home.component.ts

```diff
@@ -578,6 +612,69 @@ export class HomeComponent implements OnInit {
               // this.eventsLoading = false;
             }); */
             }
+
+            //add event markers to map
+            this.eventMarkers.addTo(this.map);
         }
     }
+
+    public clearMapFilterForm(): void {
+        // this works but will not fully clear mat-selects if they're open when the box is clicked
+        this.mapFilterForm.reset();
+    }
+
+    public submitMapFilter() {
+        let filterParams = JSON.parse(JSON.stringify(this.mapFilterForm.value));
+
+        //collect and format selected Filter Form values
+        let eventId = filterParams.eventsControl
+            ? filterParams.eventsControl.event_id
+            : '';
+        let networkIds = filterParams.networkControl
+            ? filterParams.networkControl.toString()
+            : '';
+        let sensorIds = filterParams.sensorControl
+            ? filterParams.sensorControl.toString()
+            : '';
+        let stateAbbrevs = filterParams.stateControl
+            ? filterParams.stateControl.toString()
+            : '';
+        //surveyed = true, unsurveyed = false, or leave empty
+        let surveyed = filterParams.surveyedControl
+            ? filterParams.surveyedControl
+            : '';
+        let HWMTrue = filterParams.HWMOnlyControl ? '1' : '';
+        let sensorTrue = filterParams.sensorOnlyControl ? '1' : '';
+        //Pre-deployed bracket site is HousingTypeOne=1 in API
+        let bracketTrue = filterParams.bracketSiteOnlyControl ? '1' : '';
+        let RDGTrue = filterParams.RDGOnlyControl ? '1' : '';
+        let opDefinedTrue = filterParams.OPDefinedControl ? '1' : '';
+
+        // format url params into single string
+        let urlParamString =
+            'Event=' +
+            eventId +
+            '&State=' +
+            stateAbbrevs +
+            '&SensorType=' +
+            sensorIds +
+            '&NetworkName=' +
+            networkIds +
+            '&OPDefined=' +
+            opDefinedTrue +
+            '&HWMOnly=' +
+            HWMTrue +
+            '&HWMSurveyed=' +
+            surveyed +
+            '&SensorOnly=' +
+            sensorTrue +
+            '&RDGOnly=' +
+            RDGTrue +
+            '&HousingTypeOne=' +
+            bracketTrue;
+
+        this.sitesService.getFilteredSites(urlParamString).subscribe((res) => {
+            this.mapResults(res);
+        });
+    }
```

Comment:
I think it may be better to handle the parsing of the filter selections into a URL string in the sitesService. I'd suggest forming the selections into a simple JSON object that can then be parsed out into the string. My #106 code has an example of this. But we could also just create a new issue to convert it that way.